www/include/configuration/configObject/host/DB-Func.php

```diff
@@ -94,6 +94,23 @@ function testPollerDep($instanceId)
     return true;
 }
 
+function checkIllegalChars(string $stringToTest): bool
+{
+    $centreonLog = new CentreonLog();
+
+    $centreonLog->insertLog(2, "tada");
```

Comment:
Maybe you can remove the debug

Reply:
it's still a WIP. And I still need debug for now